spec/models/miq_worker_spec.rb

```diff
@@ -1,6 +1,25 @@
 require "spec_helper"
 
 describe MiqWorker do
+  context ".corresponding_runner" do
+
+    def all_workers
+      MiqWorker.descendants.select { |c| c.subclasses.empty? }
+    end
+
+    it "finds the correct corresponding runner for workers" do
+      namespaced, legacy = all_workers.partition { |c| c.name =~ /::/ }
+
+      namespaced.each do |worker|
+        worker.corresponding_runner.should match /Runner$/
+      end
+
+      legacy.each do |worker|
+        worker.corresponding_runner.should match worker.name.slice(3..-1)
```

Comment:
Can you add a comment about the `3` magic number, that it represents removing the "Miq" prefix?